app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_count
+    author.update(posts_counter: author.posts.count)
+  end
+
+  def five_most_recent_comments
+    comments.order(created_at: :desc).limit(5)
```

Comment:
Optional -
It would be a lovely idea if you used one of the rails' active callbacks  `after_save` to call 
 the `update_post_counter` method every time a new post has been saved to the database